custom-elements/reactions/CSSStyleDeclaration.html

```diff
@@ -168,14 +168,12 @@
 }, 'A dashed property (border-width) on CSSStyleDeclaration', {propertyName: 'border-width', idlName: 'borderWidth', value1: '1px', value2: '5px'});
 
 test_mutating_style_property_value(function (instance, propertyName, idlName, value) {
-    assert_equals(idlName, 'webkitWritingMode');
-    instance.style.webkitWritingMode = value;
-}, 'A webkit prefixed camel case attribute (webkitWritingMode) on CSSStyleDeclaration', {propertyName: '-webkit-writing-mode', idlName: 'webkitWritingMode', value1: 'vertical-rl', value2: 'horizontal-tb'});
+    instance.style.webkitBorderRadius = value;
+}, 'A webkit prefixed camel case attribute (webkitBorderRadius) on CSSStyleDeclaration', {propertyName: 'border-radius', idlName: 'webkitBorderRadius', value1: '1px', value2: '5px'});
```

Comment:
Oh, `propertyName` is only used for checking the style attribute's value.
Because modern engines covert prefixed property names to unprefixed ones as they parse,
we need to be asserting against `border-radius: 1px` instead of `border-radius: 5px`.